src/translations/lang/en.json

```diff
@@ -212,7 +212,7 @@
     "DASHBOARD_ACTIONS_SWAP_ASSETS_TITLE": "Swap Assets",
     "DASHBOARD_TOKENS_TOOLTIP": "Use this area to view detailed information on the tokens in your account(s), scan for new tokens, or add tokens that aren’t on our default list.",
     "DATA": "Data",
-    "DECRYPT_ACCESS": "How would you like to access your assets? ",
+    "DECRYPT_ACCESS": "How would you like to access your assets?",
```

Comment:
ran into the same issue last time ! This is surely something we need to remove from translations

Reply:
(Talking about the wild space at the end of the string)